src/components/ProgressBar/docs/ProgressBarWiringExample.st.css

```diff
@@ -0,0 +1,7 @@
+.root {
+
+}
+
+.component {
+
+}
```

Comment:
We'll need to have an example that overrides the bar color and the bar bg color
and of course the text